superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -638,7 +638,9 @@ class SqlEditor extends React.PureComponent {
                         this.props.defaultQueryLimit,
                     )}
                   </span>
-                  <Icon name="triangle-down" />
+                  <Icons.TriangleDown
+                    iconColor={supersetTheme.colors.grayscale.base}
```

Comment:
You shouldn't access the `theme` directly from `@superset-ui/core`. You can use `useTheme` or `withTheme`.